centos/8/Dockerfile

```diff
@@ -6,14 +6,9 @@ RUN yum -y install wget yum-utils
 
 # Enable extra repositories
 RUN yum -y install epel-release
-# added PowerTools as suggested at:
-#   https://fedoraproject.org/wiki/EPEL
-RUN yum config-manager --set-enabled PowerTools
 
 # Repository for building/testing dependencies that are not present in vanilla
 # CentOS and PowerTools / EPEL repositories, e.g. some Python 2 packages
-# - fix missing locales
-ENV LC_ALL="C" LANG="en_US.UTF-8"
```

Comment:
I don't mind, but, please, ensure that tarantool builds are okay after the change.

Reply:
Right, I've created the image with the changes and checked it using packpack build with testing.